X/XDiag/build_tarballs.jl

```diff
@@ -75,8 +75,8 @@ filter!(p -> (
     (os(p) == "macos" && arch(p) == "aarch64") ||
     (os(p) == "windows" && arch(p) == "x86_64")) &&
     p.tags["julia_version"] !="1.6.3" &&
-    p.tags["julia_version"] !="1.12.0" &&
-    p.tags["julia_version"] !="1.13.0", platforms)
+    p.tags["julia_version"] !="1.13.0" &&
+    p.tags["julia_version"] !="1.14.0", platforms)
```

Comment:
Why do you filter out some Julia versions here? (Not that this will do anything right now anyway)?

Reply:
(except for filtering out 1.6.3, that will work, but not 1.13.0 and 1.14.0 which are not currently versions in `julia_versions`